### ISSUE TITLE
Base64 encoder/decoder vs compress() decompress()

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#350]
Sher Khan 
Have base64 encoder and decoder support for luaC (and mooncontroller)
Capable of storing large data into EEPROM and RAM

OgelGames 
or alternatively, compress() and decompress(): https://github.com/minetest/minetest/blob/206198431309fb00ff717f3ebd5e7c0af3de59dd/doc/lua_api.md?plain=1#L6769-L6785

**Comment by OgelGames** [comment]
Yep, this is already on the way in xLua (among other things). I haven't done much testing yet regarding performance, but limiting them is as simple as checking the input string length.
